cmd/sandbox/sandbox.go

```diff
@@ -56,7 +56,7 @@ func CreateSandboxCommand() *cobra.Command {
 			Long:  statusLong},
 		"exec": {CmdFunc: sandboxClusterExec, Aliases: []string{}, ProjectDomainNotRequired: true,
 			Short: execShort,
-			Long:  execLong, PFlagProvider: sandboxConfig.DefaultConfig},
```

Comment:
Any reason we are binding the pflagprovider

Reply:
we don't need it in exec command